docs/spec/sanitations.md

```diff
@@ -31,6 +31,142 @@ These changes are done in order to improve the overall usability, and as workaro
 - **Updated**: `"format": "datetime"`
 - **Reason**: The `date-time` format is not compatible with the OpenAPI tool. Therefore, it is updated to `datetime` to make it compatible with the tool.
 
+4. Make `"label"`,`"userEnforcedMaxToObjectIds"` properties in `"PublicAssociationDefinitionUserConfiguration"` object nullable
+
+- **Original**:
+    "PublicAssociationDefinitionUserConfiguration" : {
+        ...
+        "properties" : {
+          "userEnforcedMaxToObjectIds" : {
+            "type" : "integer",
+            "format" : "int32",
+          },
+         .
+         .
+          "label" : {
+            "type" : "string"
+          },
+          .
+          .
+    }},
+
+- **Updated**:
+     "PublicAssociationDefinitionUserConfiguration" : {
+        "properties" : {
+          "userEnforcedMaxToObjectIds" : {
+            "type" : "integer",
+            "format" : "int32",
+            "nullable": true
+          },
+         .
+         .
+          "label" : {
+            "type" : "string",
+            "nullable": true
+          },
+          .
+          .
+     }},
+
+- **Reason**: The properties `"userEnforcedMaxToObjectIds"` and `"label"` are updated to be nullable, meaning they can either hold their respective values or be null, to fix payload binding error.
+
+5. Make `"label"` property in `"PublicAssociationDefinitionCreateRequest"` object nullable
+
+- **Original**:  
+     "PublicAssociationDefinitionCreateRequest" : {
+        .
+        .
+        "properties" : {
+          .
+          .
+          "label" : {
+            "type" : "string"
+          }
+        }
+      },
+
+- **Updated**:
+     "PublicAssociationDefinitionCreateRequest" : {
+        .
+        .
+        "properties" : {
+          .
+          .
+          "label" : {
+            "type" : "string",
+            "nullable": true
+          }
+        }
+      },
+
+- **Reason**: The property `"label"` is updated to be nullable, meaning it can either hold its respective values or be null, to fix payload binding error.
+
+6. Make `"label"` property in `"AssociationSpecWithLabel"` object nullable
+
+- **Original**:  
+      "AssociationSpecWithLabel" : {
+        .
+        .
+        "properties" : {
+          .
+          .
+          "label" : {
+            "type" : "string"
+          },
+          .
+          .
+        }
+      },
+
+- **Updated**:
+      "AssociationSpecWithLabel" : {
+        .
+        .
+        "properties" : {
+          .
+          .
+          "label" : {
+            "type" : "string",
+            "nullable":true
+          },
+          .
+          .
+        }
+      },
+
+- **Reason**: The property `"label"` is updated to be nullable, meaning it can either hold its respective values or be null, to fix payload binding error.
+
+7. Make `"label"` property in `"PublicAssociationDefinitionUpdateRequest"` onject nullable
```

Comment:
```suggestion
- **Reason**: The property `label` is updated to be nullable, meaning it can either hold its respective values or be null, to fix a payload binding error.

7. Update the `label` property in `PublicAssociationDefinitionUpdateRequest` to support nullable values
```